numerics/pid.hpp

```diff
@@ -23,18 +23,21 @@ class PID {
   // Clears the state of the PID.
   void Clear();
 
-  // Adds the error between the two values to the state of the PID and
-  // returns an apparent value derived from the actual value and the control
-  // variable.
-  Value ComputeValue(Value const& apparent,
-                     Value const& actual,
+  // Adds the error between the process variable and the set-point to the state
+  // of the PID and returns an updated process variable derived from the
+  // set-point and the control variable.
+  Value ComputeValue(Value const& process_variable,
+                     Value const& set_point,
                      Time const& Δt);
 
 private:
   double const kp_;
   Inverse<Time> const ki_;
   Time const kd_;
```

Comment:
The types become:
```C++
Derivative<ControlOutput, Controlled> kp_;
Derivative<ControlOutput, Product<Controlled, Time>> ki_;
Derivative<ControlOutput, Variation<Controlled>> kd_;
```

Reply:
I don't think these are derivatives, they are just quotients.  There is no derivation in the canonical PID formula.

Reply:
OK, but note that then you need `Quotient<ControlOutput, Difference<Controlled>>`.

Reply:
Using `Normed` as discussed.